tests/FieldAlias.php

```diff
@@ -66,6 +66,18 @@ public function testLegacyOrderBy()
         $query = $mapper->where(['number' => 2])->order(['date_created' => 'ASC'])->noQuote();
         $this->assertContains("ORDER BY test_legacy." . self::$legacyTable->getDateCreatedColumnName() . " ASC", $query->toSql());
     }
+    
+    // Ordering by function
+    public function testLegacyOrderByFunction()
+    {
+        $mapper = test_spot_mapper('SpotTest\Entity\Legacy');
+        // Currently it works with mysql only
+        if (!$mapper->connectionIs('mysql')) {
```

Comment:
Is there some generic ANSI-SQL function you can use instead that will work on all supported/tested database types?